spec/models/course/group_user_spec.rb

```diff
@@ -24,14 +24,5 @@
 
       it { is_expected.not_to be_valid }
     end
-
-    describe '.managers' do
```

Comment:
Yeah, the rest of the spec can be simplified as well, since a fresh course is created for each when we do `create(:course_group)` and `create(:course_user)`.